conduct/conduct_enforcement.md

```diff
@@ -0,0 +1,206 @@
+
+* Person One foo@email (Chair of the Committee)
+* ...
+* Person Five bar@email
+
+The Jupyter Steering Council will approve the members of this five person
+committee. The members will be drawn from our entire community, and we intend
+that it *not* be only Steering Council members: anyone who is actively engaged
+with Jupyter can propose to join this committee (we'll explicitly solicit
+volunteers when short of five members).  One member will be designated chair of
+the group and will be responsible for all reports back to the Steering Council.
+
+
+## Enforcement guidelines and principles
+
+Enforcing the Code of Conduct impacts our community today and for the future.
+It's an action that we do not take lightly. When reviewing enforcement
+measures, the Code of Conduct Committee will keep the following values and
+guidelines in mind:
+
+* Act in a personal manner rather than impersonal.  The Committee can engage
+  the parties to understand the situation, while respecting the privacy and any
+  necessary confidentiality of reporters.  However, sometimes it is necessary
+  to communicate with one or more individuals directly: the Committee's goal is
+  to improve the health of our community rather than only produce a formal
+  decision.
+
+* Emphasize empathy for individuals rather than judging behavior, avoiding
+  binary labels of "good" and "bad/evil". Overt, clear-cut agression and
+  harrassment exists and will be addressed unambiguously.  But many scenarios
+  that can prove challenging to resolve are those where normal disagreements
+  devolve into inappropriate behavior from multiple parties.  Understanding the
+  full context and finding a path that re-engages all is hard, but ultimately
+  the most productive for our community.
+
+* Help increase engagement in good discussion practice: try to identify where
+  discussion may have broken down and provide actionable information, pointers
+  and resources that can help enact positive change on these points.
+
+* Be mindful of the needs of new members: provide them with explicit support
+  and consideration, with the aim of increasing participation from
+  underrepresented groups in particular.
+
+* Individuals come from different cultural backgrounds and native languages.
+  While lack of intent to harm is not an excuse, try to identify any honest
+  misunderstandings caused by a non-native speaker and help them understand the
+  issue and how to change.  Complex discussion in a foreign language can be
+  very intimidating, and we want to grow our diversity also across
+  nationalities and cultures.
+
+* Our actions will reflect compassion for all individuals. We will seek to
+  understand, to educate, and, as necessary, take action.
+
+
+**Mediation:** voluntary, informal mediation is a tool at our disposal.  In
```

Comment:
While I see that from a purely "operational" standpoint it may fit better in one of those other two, it still seems to me that in terms of gathering the principles and general approach the Committee should use, it actually works well where it currently is.  Basically that section is intended to frame the "mindset" for the Committee members, including all options and ideas they should keep in mind for productive outcomes.  In that sense, I see Mediation as one such idea, even if it has a more operational flavor than the others.  At first I had it as a bullet like the others, but it got too long, which is why I broke it up.

Does this make sense?  I see your point, but I toyed with the alternatives and still prefer the current version...